Magento_Downloadable/web/js/downloadable-mixin.js

```diff
@@ -0,0 +1,59 @@
+define([
+    'jquery',
+    'jquery-ui-modules/widget',
+    'Magento_Catalog/js/price-box'
+], function ($) {
+    'use strict';
+
+  return function (widget) {
+    /**
+     * Downloadable widget
```

Comment:
Change this to: `Downloadable widget mixin` + I think we should start to add some short info what is the purpose of this mixin/what and was changed/add. Comparing original file the changes are very small but almost the whole original file code is in here. It will be good to left info for someone who will in future look into this file